src/diffusers/loaders.py

```diff
@@ -1501,6 +1509,7 @@ def load_lora_into_text_encoder(
         lora_scale=1.0,
         low_cpu_mem_usage=None,
         _pipeline=None,
+        adapter_name=None,
```

Comment:
```suggestion
        adapter_name=None,
        _pipeline=None,
```

Let's put the private variable one below

Reply:
addressed in the latest commit